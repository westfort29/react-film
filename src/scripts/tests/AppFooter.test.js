import React from 'react';
import { AppFooter } from '../';
import renderer from 'react-test-renderer';

test('AppFooter should contain right data', () => {
  const component = renderer.create(
    <AppFooter />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

