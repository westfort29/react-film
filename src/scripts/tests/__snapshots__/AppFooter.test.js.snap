// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppFooter should contain right data 1`] = `
<footer
  className="app-footer"
>
  <ul
    className="app-footer__info-list"
  >
    <li
      className="app-footer__info-item"
    >
      This is footer
    </li>
    <li
      className="app-footer__info-item"
    >
      Copyright things
    </li>
  </ul>
</footer>
`;
